ui/app/registration/views/patientAction.html

```diff
@@ -31,4 +31,6 @@
     </span>
     <button type="submit" class="right submit-btn" ng-click="setSubmitSource('save')"
             accesskey="{{ ::'REGISTRATION_SAVE_ACCESS_KEY' | translate}}" ng-bind-html="::'REGISTRATION_LABEL_SAVE' | translate"></button>
+    <span class="right" style="margin-right:10px" ng-click="sendWhatsAppMessage()"  ng-show="enableWhatsAppButton" ><img  src="https://image.freepik.com/free-vector/whatsapp-icon-design_23-2147900927.jpg"  height="35" size="35">
```

Comment:
you are referencing a freepik asset (with their watermarks). You need to download the PNG (free without watermakrs) and [package with the application under images](https://github.com/Bahmni/openmrs-module-bahmniapps/tree/master/ui/app/images)

Reply:
You can download an image from here: https://commons.wikimedia.org/wiki/File:WhatsApp.svg

Reply:
Added the image as suggested above in images folder.Please review.